python/cudf/udf_cpp/CMakeLists.txt

```diff
@@ -26,12 +26,27 @@ rapids_find_package(
   INSTALL_EXPORT_SET udf-exports
 )
 
+# The below in vendored/inspired by legate dataframe
+execute_process(
+  COMMAND "python" -c "from numba.cuda.runtime.nrt import get_include; print(get_include())"
+  OUTPUT_VARIABLE NRT_INCLUDE_DIR
+  ERROR_VARIABLE NRT_ERROR
+  RESULT_VARIABLE NRT_RESULT
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+if(${NRT_RESULT})
+  message(FATAL_ERROR "Error while trying to obtain NRT include directory:\n${NRT_ERROR}")
```

Comment:
Can we say "Numba Runtime" or "numba-cuda runtime" or something besides NRT? This error might be cryptic to anyone who doesn't know Numba internals.